src/ClusterManager/job_manager.py

```diff
@@ -601,7 +617,7 @@ def get_jobs_info(jobs):
 
 def mark_schedulable_non_preemptable_jobs(jobs_info, cluster_schedulable,
                                           vc_schedulables):
-    stop_scheduling = None
+    stop_schedulings = {} # vc_name -> the first blocking job of this vc
```

Comment:
We need a way to configure this policy for a specific VC.

Reply:
yes, working